dockers/docker-platform-monitor/start.sh.j2

```diff
@@ -48,3 +48,6 @@ supervisorctl start xcvrd
 {% if not skip_psud %}
 supervisorctl start psud
 {% endif %}
+
+post-syseeprom -w
```

Comment:
Did you handle error code?

Reply:
I added error codes and error log for all failed case in the scripts. 
Here I am thinking that if the scripts failed, then the reason could be due to failed EEPROM or have some issue with DB, my view is that in these cases we can do nothing here, the error log printed out would be enough to indicate the failure reason.  thoughts?

Reply:
Still applicable.

---
In reply to: [285325002](https://github.com/Azure/sonic-buildimage/pull/2866#discussion_r285325002) [](ancestors = 285325002)

Reply:
It's always be a good practice to report error code on each layer. If you could exit with error code, and supervisord will catch it and mark the job state. It is not needed in this use case, but still recommended.

---
In reply to: [285457447](https://github.com/Azure/sonic-buildimage/pull/2866#discussion_r285457447) [](ancestors = 285457447)

Reply:
> It's always be a good practice to report error code on each layer. If you could exit with error code, and supervisord will catch it and mark the job state. It is not needed in this use case, but still recommended.
> 
> In reply to: [285457447](https://github.com/Azure/sonic-buildimage/pull/2866#discussion_r285457447) [](ancestors = 285457447)

ok, added error code handling